nixos/modules/services/networking/ntp/chrony.nix

```diff
@@ -12,11 +12,14 @@ let
   rtcFile = "${stateDir}/chrony.rtc";
 
   configFile = pkgs.writeText "chrony.conf" ''
-    ${concatMapStringsSep "\n" (server: "server " + server + " " + cfg.serverOption + optionalString (cfg.enableNTS) " nts") cfg.servers}
-
-    ${optionalString
-      (cfg.initstepslew.enabled && (cfg.servers != []))
-      "initstepslew ${toString cfg.initstepslew.threshold} ${concatStringsSep " " cfg.servers}"
+    ${concatMapStringsSep "\n" (
+      server: "server " + server + " " + cfg.serverOption + optionalString (cfg.enableNTS) " nts"
```

Comment:
`nts` is also a `serverOption`. If `serverOption` is a list of strings, `enableNTS` is redundant. If `serverOption` is an enum, `enableNTS` is strange because `nts` is a per-server option in chrony but we treat it as a "global" option (i.e. it will apply to either all servers or no servers)